libraries/chain/transaction_context.cpp

```diff
@@ -52,7 +52,7 @@ namespace eosio { namespace chain {
    ,trx(t)
    ,id(trx_id)
    ,undo_session()
-   ,trace(std::make_shared<transaction_trace>())
+   ,trace(std::make_unique<transaction_trace>())
```

Comment:
Did you intend to also make the `trace` member variable in `transaction_context` a `unique_ptr`?

Reply:
oops, I thought I reverted that.